visitors/es6-template-visitors.js

```diff
@@ -46,9 +46,12 @@ function visitTemplateLiteral(traverse, node, path, state) {
       // Concatenat adjacent substitutions, e.g. `${x}${y}`. Empty templates
       // appear before the first and after the last element - nothing to add in
       // those cases.
+      if (ii === 0) {
+        utils.append('"" + ', state);
+      }
       if (ii > 0 && !templateElement.tail) {
         // + between substitution and substitution
-        utils.append(' + ', state);
+        utils.append(' + "" + ', state);
```

Comment:
Yea, I bet we don't. I did this first before always putting the `""` at the beginning and then I wrote more tests :). I'll remove it.